Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-main.view.html

```diff
@@ -99,28 +99,47 @@ <h4 id="systemUsageHeaderName">{{systemUsageName}}</h4>
                            data-field="expirationDate">
                 </div>
                 <div class="form-group">
-                    
-                    <select-simple data-id="lifeCycleStatus"
-                                   data-label="Status"
-                                   data-options="lifeCycleStatusOptions"
-                                   data-ng-model="usageViewModel.lifeCycleStatus"
-                                   data-ng-disabled="!hasWriteAccess"
-                                   data-auto-save-url="{{::autoSaveUrl}}"
-                                   data-field="lifeCycleStatus">
-                    </select-simple>
+                    <label for="lifeCycleStatus" class="control-label small">Livscyklus</label>
+                    <single-select-select2 id="lifeCycleStatus" ng-model="lifeCycleStatusModel" />
+                </div>
+                <div ng-if="validationStatus">
+                    <label class="control-label small">Status</label>
+                    <!-- Show the simple version of the status if there are no details to explain-->
+                    <div class="form-group object-status-active-summary" ng-if="validationStatus.valid && validationStatus.errors.length === 0">
+                        <!--TODO: Change text to danish-->
+                        <span ng-if="validationStatus.valid">Systemet er <strong>aktivt</strong></span>
+                    </div>
+                    <!-- Show the detailed summary if any errors are involved at all-->
+                    <div class="form-group object-status-active-summary" ng-if="validationStatus.errors.length > 0">
```

Comment:
I did the updates. but you have included a whole lot of mess that does not belong to it-system validations here.. it-systems are either valid or not. They cannot be valid if there are errors, so again.. too much copy paste programming and remember what happens when you do that... 😢